rpc/src/v1/types/mod.rs

```diff
@@ -16,17 +16,18 @@
 
 mod action;
 mod block;
+mod mem_pool;
 mod transaction;
 mod unsigned_transaction;
 mod work;
 
 pub use self::action::{Action, ActionWithTracker};
 pub use self::block::Block;
 pub use self::block::BlockNumberAndHash;
+pub use self::mem_pool::MemPoolMinFees;
 pub use self::transaction::{PendingTransactions, Transaction};
 pub use self::unsigned_transaction::UnsignedTransaction;
 pub use self::work::Work;
-
```

Comment:
https://github.com/kodebox-io/kodebox-internal/wiki/Tips-for-Developers#selectively-stage-your-work